src/wallet/rpcwallet.cpp

```diff
@@ -768,8 +770,10 @@ UniValue getbalance(const JSONRPCRequest& request)
 
     LOCK2(cs_main, pwallet->cs_wallet);
 
-    if (request.params.size() == 0)
-        return  ValueFromAmount(pwallet->GetBalance());
+    bool ignoreUnspendable = request.params[3].isNull() || !request.params[3].get_bool();
```

Comment:
I find it confusing that the API uses `include_unspendable`, but internally you use `ignoreUnspendable` which is the inverse. Any reason not to be consistent?